hieradata/common.yaml

```diff
@@ -290,6 +290,19 @@ azure::k8s::management_ssh_privkey: |
 azure::k8s::management_ssh_pubkey: 'AAAAB3NzaC1yc2EAAAADAQABAAABAQDvly3ZF9ntYXlvw/SS/v+8LwGten0CHvYdIjY1y1NcR2SrX3Azv/zDPStTaiRTcOWGEByn84q6VRyH4+iN+HY31tMopVGpKEUJvIkP7qpW8UGpNS5MPOhhCfOsBxGHZ6AzjLwpOsHrlEahgl7tU1JW4VnX4jRy+2mQJkqnHtPeqYZMtNZwymhp8NjvlyxtxLw5pNu5wd8unXeuLkH73zJvk8kNgjw5P2cg4UBVXXQEdyRJ0rwLw4mLVzHvY2sIz5T9AN+qz29VgN6mm1fZPe+sHLfDLBFK5AWlvTv0jvX+TyrfryG4XwnU/kwROn5KJDa1SnuAq78JloZOsg24NXaB'
 
 
+profile::datadog_ssl_check::sites:
+    - issues.jenkins-ci.org
+    - wiki.jenkins-ci.org
+    - jenkins.io
+    - www.jenkins.io
+    - ci.jenkins.io
+    - mirrors.jenkins.io
+    - accounts.jenkins.io
+    - usage.jenkins.io
+    - pkg.jenkins.io
+    - updates.jenkins.io
+    - plugins.jenkins.io
```

Comment:
rating.jenkins.io is another.

Minor: Alphabetical order would probably be nicer.